packages/ag-charts-community/src/components/toolbar/floatingToolbar.ts

```diff
@@ -142,6 +151,21 @@ export abstract class FloatingToolbar<
         const buttonBounds = this.getButtonBounds().map(
             (bounds) => new BBox(bounds.x + popoverBounds.x, bounds.y + popoverBounds.y, bounds.width, bounds.height)
         );
-        this.onToolbarMoved({ popoverBounds, buttonBounds });
+        this.events.dispatch('toolbar-moved', { popoverBounds, buttonBounds });
+    }
+
+    private createDragHandle() {
+        const { popover } = this;
+
+        const dragHandle = new NativeWidget<HTMLElement>(
+            createElement('div', 'ag-charts-floating-toolbar__drag-handle')
+        );
+        dragHandle.getElement().innerHTML = `<span class="${getIconClassNames('drag-handle')} ag-charts-toolbar__icon"></span>`;
+        dragHandle.getElement().addEventListener('mousedown', (event) => {
+            popover.startDragging(event, dragHandle.getElement());
+        });
+        dragHandle.getElement().title = this.ctx.localeManager.t('toolbarAnnotationsDragHandle');
```

Comment:
I'll look into creating widgets/components for drag handles and icons in the future.